apps/passport-client/src/worker/service-worker.ts

```diff
@@ -1,6 +1,12 @@
 // this file is loaded as a service worker
+import { ARTIFACTS_NPM_VERSION } from "@pcd/gpcircuits/constants";
 import { setTimeout as promiseTimeout } from "isomorphic-timers-promises";
-import { SERVICE_WORKER_ENABLED } from "../sharedConstants";
+import {
+  SERVICE_WORKER_ENABLED,
+  ZUPASS_GPC_ARTIFACT_PATH
+} from "../sharedConstants";
+
+const GPC_ARTIFACTS_NPM_VERSION = ARTIFACTS_NPM_VERSION;
```

Comment:
Can this be handled in the import?  `import { ARTIFACTS_NPM_VERSION as GPC_ARTIFACTS_NPM_VERSION } ...`?